wsi/drm/drmdisplay.cpp

```diff
@@ -162,16 +162,36 @@ bool DrmDisplay::GetDisplayAttribute(uint32_t config /*config*/,
 
   float refresh;
   bool status = true;
+
+  // Check if different resolution is configured
+  bool custom_resolution = IsCustomResolution();
```

Comment:
DrmDisplay is inherited from PhysicalDisplay and should have access to all the protected variables. I think we can just use the variables directly here ?

Reply:
Technically yes, but accessing base class variable directly instead of through methods is not something I would prefer. It helps in debugging as well.